projects/1.20/assets/rats/rats/lang/zh_cn.json

```
@@ -0,0 +1,1028 @@
+{
+  "advancement.ratlantis.defeat_rat_baron.desc": "击败老鼠男爵",
+  "advancement.ratlantis.defeat_rat_baron.title": "死亡或荣耀",
+  "advancement.ratlantis.dutchrat_wheel.desc": "击败飞翔的荷兰鼠",
+  "advancement.ratlantis.dutchrat_wheel.title": "LEDLE LEDLE LEDLE LEEEE",
+  "advancement.ratlantis.feral_ratlantean.desc": "遇到鼠西洲过去的高贵居民如今的残态",
+  "advancement.ratlantis.feral_ratlantean.title": "追本溯源",
+  "advancement.ratlantis.gem_of_ratlantis.desc": "用鼠套花瓣和绿宝石来制作鼠西洲宝石",
+  "advancement.ratlantis.gem_of_ratlantis.title": "近乎圣洁",
+  "advancement.ratlantis.ghost_pirat.desc": "遭遇幽灵鼠盗，久逝水手的被诅咒的灵魂",
+  "advancement.ratlantis.ghost_pirat.title": "吓我一跳",
+  "advancement.ratlantis.marbled_cheese_golem.desc": "击败鼠西族战机",
+  "advancement.ratlantis.marbled_cheese_golem.title": "人机对决（或鼠机对决）",
+  "advancement.ratlantis.marbled_cheese_golem_core.desc": "合成鼠西族战机核心！现在，将鼠西族战机头颅放在顶部，并用大理石纹奶酪包裹核心！（任意大理石纹奶酪，做成铁傀儡的形状）",
+  "advancement.ratlantis.marbled_cheese_golem_core.title": "我们可以重构……",
+  "advancement.ratlantis.neoratlantean.desc": "击败新·鼠西族",
+  "advancement.ratlantis.neoratlantean.title": "但伙计，我数学不好吗",
+  "advancement.ratlantis.oratchalcum_ingot.desc": "获得鼠铜锭",
+  "advancement.ratlantis.oratchalcum_ingot.title": "感受力量！",
+  "advancement.ratlantis.pirat.desc": "遇到鼠盗，它是一只放荡不羁、放浪形骸的老鼠",
+  "advancement.ratlantis.pirat.title": "准备好了吗？孩子们",
+  "advancement.ratlantis.rat_upgrade_archeologist.desc": "合成老鼠升级：考古学家，鼠西洲的备选方案",
+  "advancement.ratlantis.rat_upgrade_archeologist.title": "温新知故",
+  "advancement.ratlantis.rat_upgrade_buccaneer.desc": "合成老鼠升级：海盗，为老鼠提供了用不着的16世纪武装",
+  "advancement.ratlantis.rat_upgrade_buccaneer.title": "是的船长！",
+  "advancement.ratlantis.rat_upgrade_mount_biplane.desc": "合成老鼠升级：双翼机坐骑，然后获得你自己的王牌飞行员老鼠",
+  "advancement.ratlantis.rat_upgrade_mount_biplane.title": "霍鼠德·休斯",
+  "advancement.ratlantis.rat_upgrade_nonbeliever.desc": "合成老鼠升级：非信者，可以完爆一切的老鼠升级",
+  "advancement.ratlantis.rat_upgrade_nonbeliever.title": "对于非信者来说战神算什么？",
+  "advancement.ratlantis.rat_upgrade_psychic.desc": "合成老鼠升级：灵媒，并且拥有自己的通灵鼠",
+  "advancement.ratlantis.rat_upgrade_psychic.title": "才不是U盘",
+  "advancement.ratlantis.rat_upgrade_ratinator.desc": "合成老鼠升级：终结鼠，将致命的赛博老鼠变为现实",
+  "advancement.ratlantis.rat_upgrade_ratinator.title": "如同泪水消失在雨中",
+  "advancement.ratlantis.ratglove_petals.desc": "利用九个鼠套花合成鼠套花瓣",
+  "advancement.ratlantis.ratglove_petals.title": "花的力量",
+  "advancement.ratlantis.ratlantean_ratbot.desc": "遭遇鼠西族机器鼠，旧时代的遗物。",
+  "advancement.ratlantis.ratlantean_ratbot.title": "不锈钢老鼠",
+  "advancement.ratlantis.ratlantean_spirit.desc": "遇到湮没已久的鼠西族鬼魂",
+  "advancement.ratlantis.ratlantean_spirit.title": "一兆死灵遗灰",
+  "advancement.ratlantis.ratlantis.desc": "使用肥厚奶酪令牌在地上随处创建通往鼠西州的入口。并通过入口进入！",
+  "advancement.ratlantis.ratlantis.title": "鼠西洲！",
+  "advancement.ratlantis.ratlantis_armor.desc": "获得一整套的鼠西族战甲",
```

```suggestion
  "advancement.ratlantis.ratlantis_armor.desc": "获得一整套的鼠西族盔甲",
```
armor 盔甲、护甲值，非护甲值时均译作“盔甲”